packages/shared/lib/sdk/sync.ts

```diff
@@ -437,8 +447,9 @@ export class NangoAction {
                     msg: content
                 }
             });
+
             if (response.status > 299) {
-                console.log(
+                console.error(
```

Comment:
I think I have copy-pasted this everywhere it is using the persistAPI so you might want to fix it also there

Reply:
Updated with https://github.com/NangoHQ/nango/pull/1601/commits/da1f2d1b684fcf484a4640d7cd2f27cc7cdefbb7